cloudshell/rest/async_api.py

```diff
@@ -0,0 +1,152 @@
+from pathlib import Path
+from typing import Union, BinaryIO, List, Optional
+from urllib.parse import urljoin
+
+import aiofiles
+import aiohttp
+from aiohttp import ClientSession
+from async_property import async_cached_property
+
+from cloudshell.rest.exceptions import (
+    FeatureUnavailable,
+    PackagingRestApiError,
+    ShellNotFoundException,
+)
+
+
+class AsyncPackagingRestApiClient:
```

Comment:
does this work in py3? shouldn't it be like class AsyncPackagingRestApiClient(object):?

Reply:
Yes, all classes in Python3 are based on the object and you don't need to add it

Reply:
great :)